slu/slu/dev/prepare.py

```diff
@@ -21,6 +21,9 @@ def preprocess(df):
         try:
             alternatives = data[const.ALTERNATIVES]
             data = normalize(alternatives)
+            # ignore cases of type {"alternatives": [{"transcript": "<UNK>"}]}
+            if all(alt in const.TEXTS_TO_IGNORE for alt in data): 
```

Comment:
We should also be looking for a series of `"<UNK>"`.

Aside: Do we have any inputs on filtering the alternative vs filtering the data point?

Reply:
> We should also be looking for a series of "<UNK>".

Yes, but imo it would be better to add all the combinations to texts_to_ignore rather than processing each transcript. I've added only `<unk>` for the moment.

Alternative is filtered only when all transcripts are bad atm. No other rigid criteria.

Reply:
> it would be better to add all the combinations to texts_to_ignore
🤔 I have seen `"<UNK> <UNK> <UNK>"` but it is unlikely that `all` transcripts would have that pattern.

I think the logic works for this pass but it might be worth exploring to see if filtering the `"<UNK>"` transcript help instead of the whole utterance.